apps/web/src/components/note/index.tsx

```diff
@@ -403,6 +405,16 @@ export const noteMenuItems: (
       onClick: () => store.archive(!note.archived, ...ids),
       multiSelect: true
     },
+    {
+      type: "button",
+      key: "spellcheck",
+      title: strings.spellCheck(),
+      isChecked: note.spellcheck,
+      icon: SpellCheck.path,
+      isHidden: IS_DESKTOP_APP && !useSpellChecker.getState().enabled,
+      onClick: () => store.spellcheck(!note.spellcheck, ...ids),
+      multiSelect: true
+    },
```

Comment:
I think this setting should only be available from inside the editor. Maybe in the status bar or in the properties panel. No point putting this here as it'll probably only be used via the editor.